citi-onboarding/valepcd
# Feature(footer): create component

***What I Did***
- Create Footer component
- Stylize component
- Import Footer on `rindex.jsx`

***How To Test***
1. Just run the front-end and check if the component is rendered. 
2. Test the responsiveness.
